src/server/conn.rs

```diff
@@ -250,6 +251,9 @@ impl Conn {
         }
 
         // no data for writing, remove writable
+        if self.send_buffer.capacity() > SEND_BUFFER_SHRINK_THRESHOLD {
```

Comment:
can we add a configuration for test coverage, and the store connection can use a bigger threshold.